src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Tab } from '@mui/material';
+import Tabs from '@mui/material/Tabs';
+import React from 'react';
```

Comment:
I've read there is no need to import React malually
https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj
If I'm not right I would be glad to hear the explanation

Reply:
fixed